data/json/requirements/toolsets.json

```diff
@@ -1,5 +1,23 @@
 [
   {
+    "id": "cooking_standard",
+    "type": "requirement",
+    "//": "Cooking of standard meals",
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [
+      [ "fire", -1 ],
+      [ "hotplate", 10 ],
+      [ "mess_kit", 10 ],
+      [ "mil_mess_kit", 10 ],
+      [ "oil_cooker", 10 ],
+      [ "survivor_mess_kit", 10 ],
+      [ "gasoline_cooker", 10 ],
+      [ "charcoal_cooker", 10 ],
+      [ "popcan_stove", 10 ],
+      [ "toolset", 10 ]
+    ] ]
+  },
```

Comment:
> Conversion from electricity to heat is nearly perfect. Specifically for heat sources, the efficiency shouldn't increase, only decrease from the baseline

Agreed

> At the very least, the multiplier next to hotplates should be 1.

This confounds `efficiency (%)` with `energy (kWh)`. One unit of `battery` doesn't have the same kWh as `lamp_oil` or `gasoline`. This currently doesn't mean much but it is worth considering the possible later effects.

> Some other cookers can be worse. Most notably the oil cookers

Yes, they are very generous. It would add some variety to have the differing fuels have different characteristics. At the moment all our stoves are just more of the same.

> 4-5 would be tolerable.

I chose 4 because it has the most divisors and at 250ml/vol results in 10 charges/L so if we ever do scale volumetric units the later maths won't be horrible.

> Until we have a proper requirement for "keep fire alive through the whole crafting process"

I'm looking into this now, not sure how hard it may or may not be to fix.